translator/tocwconfig/sampleConfig/complete_linux_config.yaml

```diff
@@ -314,6 +362,8 @@ service:
                 - awscloudwatch
             processors:
                 - ec2tagger
+                - filter/metrics/0
```

Comment:
These each need their own pipeline. We can't have the filters in the same pipeline in case 1 jmx allows jvm and another allows hadoop.

Reply:
These filters look like they're in the wrong pipeline. Why is the telegraf_mem receiver part of it?